working-groups/security/disclosure.md

```diff
@@ -0,0 +1,101 @@
+# Early Disclosure of Security Vulnerabilities
+
+(This policy is based on the Istio Early Disclosure Policy at https://github.com/istio/community/blob/master/EARLY-DISCLOSURE.md).
```

Comment:
nit, since this may change, should we point to the actual point in time? like the commit point?